<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
   <item row="0" column="0">
    <layout class="QGridLayout" name="titleLayout" rowstretch="0" columnstretch="0">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="title">
         <property name="styleSheet">
          <string notr="true">font-size: 24px; font-weight: 900; border-radius: 10px;</string>
         </property>
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,1,111" columnstretch="0,1,8,1,1">
     <item row="2" column="2" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="resultLabel">
       <property name="styleSheet">
        <string notr="true">font-size: 16px; font-weight: 600; border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Результаты по запросу:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="searchLine">
       <property name="styleSheet">
        <string notr="true">padding: 10px;
font-size: 14px;
border: 2px solid #ccc;
border-radius: 10px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="4">
      <widget class="QTextBrowser" name="searchBrowser"/>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="searchButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
                padding: 10px 20px;
                font-size: 14px;
                background-color: #007bff;
                color: white;
                border: none;
                border-radius: 5px;
            }
</string>
       </property>
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
